qa/integration-tests/src/test/java/io/zeebe/broker/it/health/BrokerMonitoirngEndpointTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it.health;
+
+import static io.restassured.RestAssured.given;
+
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.zeebe.containers.ZeebeBrokerContainer;
+import java.io.IOException;
+import org.hamcrest.CoreMatchers;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public final class BrokerMonitoirngEndpointTest {
+
+  static ZeebeBrokerContainer sutBroker;
+
+  static RequestSpecification brokerServerSpec;
```

Comment:
I'm not familiar with this framework `RestAssured`, but I like it, great stuff :+1: 

Maybe when we wrap up the OKR and we show it off, you can also mention how to test these things, as I don't know if everybody on the team is familiar with this.